ui/src/app/pages/workspace/workspace-about.tsx

```diff
@@ -488,7 +530,7 @@ export const WorkspaceAbout = fp.flow(
             </div>
 
             <TooltipTrigger
-              content='Only workspaces owners can view the billing report'
+              content='Only workspaces owners can view the billing report.'
```

Comment:
workspaces -> workspace

Reply:
thanks @jmthibault79 ...updated in https://github.com/all-of-us/workbench/pull/8658

Reply:
Thanks!  All looks good there